src/main/java/eu/assuremoss/VulnRepairDriver.java

```diff
@@ -46,4 +80,17 @@ public static void main( String[] args )
             // TODO: print out patches to files or pass them to a Visualizer
         }
     }
+
+    public static String getProjectName() {
```

Comment:
Delete these static getters and make the static fields they return public, so that other classes can access them directly.